### DISCUSSION TITLE
Copilot: 'Node.js exited with status 126'

### DISCUSSION THREAD

**Discussion by taufek** [community/community#52418] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I recently installed `copilot.vim` plugin on my Neovim but each time I run any Copilot command (e.g. `:Copilot status`), it will say "Copilot: 'Node.js exited with status 126'".  Appreciate for any help to get Copilot running properly. Below is my setup:

- Neovim 0.8.2
- Node 14.9.0 (installed using asdf package manager and I've tried with other version such as 16.x, 17.x and 18.x still getting same issue).

**Comment by taufek** [comment, marked as answer]
Found out I that I need to set node version on my home directory than Copilot started to work as expected.

**Reply by bonzofenix** [reply]
hero

**Comment by jeroenbourgois** [comment]
@taufek sry to seem dumb, but _how_ did you set the node version? We use `asdf` too so the version differs depending on the project directory.

**Reply by jastkand** [reply]
@jeroenbourgois if still relevant, you can use the following to set the node version on home directory
```
cd ~
asdf local nodejs 20.6.1 // whatever node version you want it to use
```

Also you can put the

```
let g:copilot_node_command = "/Users/<user_name>/.asdf/shims/node"
```

to your `.vimrc`, but it wasn't required in my case.

**Reply by mraaroncruz** [reply]
I just had this issue and this helped but I just set my node version to global, I guess I had forgotten to do it or assumed it happened default when you only have one version.

```
asdf global nodejs 20.6.1
```
